src/main/resources/theme-resources/messages/messages_fr.properties

```diff
@@ -1,10 +1,9 @@
-authCodeText=Votre code � usage unique est %1$s et est valable pour %2$d minutes.
-
-smsAuthTitle=Code SMS
-smsAuthCodeTitle=SMS Code
-smsAuthLabel=Code
-smsAuthInstruction=Entrez le code que nous avons envoy� � votre appareil.
+authCodeText=%1$s is your one time passcode from OHCRN. %1$s est votre code d'acc�s � usage unique de l'OHCRN.
 
+smsAuthTitle=V�rification en 2 �tapes
```

Comment:
i guess the encoding in a diff is different than in the codebase? the accented characters show correctly in github files and the browser. The messages.properties files default to UTF-8 encoding.